frontend/src/app/components/Header.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
 import { Link } from 'react-router';
-import classnames from 'classnames';
 
 import LayoutWrapper from './LayoutWrapper';
 import RetireConfirmationDialog from './RetireConfirmationDialog';
+import Settings from './Settings';
 
 export default class Header extends React.Component {
 
   constructor(props) {
     super(props);
+
     this.closeTimer = null;
+    this.showSettingsMenu = this.showSettingsMenu.bind(this);
```

Comment:
But perhaps I should leave it for the sake of consistency with other components?